core/query/query.ml

```diff
@@ -399,7 +554,7 @@ struct
         match Value.Env.lookup var val_env, Env.Int.find exp_env var with
         | None, Some v -> v
         | Some v, None -> expression_of_value v
-        | Some _, Some v -> v (*eval_error "Variable %d bound twice" var*)
+        | Some _, Some v -> v (*query_error "Variable %d bound twice" var*)
```

Comment:
Does this comment still need to exist?  It seems we are just using the exp_env version now shadowing the val_env occurrence.

Reply:
TBH I don't know how this would happen and on what grounds we ignore the val_env version.
It looks like this was previously considered an error, but now it's not.
I updated the comment because the original `eval_error` has now become a more generic `query_error`. If the consensus is that the comment is now useless, I'll remove it.

Reply:
OK, let's leave as is for now unless someone chimes in.